main.py

```diff
@@ -668,6 +748,19 @@ def _send_flow_mods_from_request(self, dpid, command, flows_dict=None):
         except SwitchNotConnectedError as error:
             raise FailedDependency(str(error))
 
+    @classmethod
+    def build_flow_mod_from_command(cls, flow, command):
```

Comment:
```suggestion
    @staticmethod
    def build_flow_mod_from_command(flow, command):
```

Reply:
Thanks for this suggestion, @ajoaoff, I've just committed it.